patches/common/chromium/browser_compositor_mac.patch

```diff
@@ -29,7 +29,7 @@ diff --git a/content/browser/renderer_host/browser_compositor_view_mac.mm b/cont
 index 92afcc77910610e53378f55adc003cc1bdf3109a..42bd6fd7c169de36c775471c68b456f1386ff666 100644
 --- a/content/browser/renderer_host/browser_compositor_view_mac.mm
 +++ b/content/browser/renderer_host/browser_compositor_view_mac.mm
-@@ -81,6 +81,12 @@ BrowserCompositorMac::~BrowserCompositorMac() {
+@@ -81,6 +81,12 @@
```

Comment:
It's weird that your `git` dropped the function annotation here. Do you have something in your `~/.gitconfig` that's doing that? can we override it in `git-export-patches` so that we get consistent formatting?

Reply:
@nornagon My git config only has the line endings config and my name/email.  Nothing fancy there 🤔

Reply:
anything in `.git/config` then?

Reply:
Resolved through #17598